Game.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import Block from './Block';
+
+function Game() {
+
+  const [isXcurrentPlayer, setNextPlayer] = React.useState(true);
+
+    function changePlayer(){
+    setNextPlayer(prevState => {
+      if(prevState) return false; 
+      else return true;
+        })
+    }
+
+    function draw(e){        
```

Comment:
this is handler, so name and parameters of this function can be better, this is more intuitive for example:

const onPlayerMove = (event) => { ... }